README.md

```diff
@@ -207,17 +201,12 @@ docker compose run --rm web bundle exec rake db:migrate
 
 ## Dev in Docker
 
-For dev in docker add `SUPERUSER`.
-```
-# docker compose exec --user postgres db psql -c "ALTER USER planner WITH SUPERUSER;"
-```
-
 To reset the instance
 ```
 docker compose down
-docker compose up -d db
-docker compose exec --user postgres db psql -c "DROP DATABASE planner;"
-docker compose exec --user postgres db psql -c "DROP ROLE planner;"
+docker volume rm pg_data
```

Comment:
Better
```
docker compose down -v
```

Reply:
It is better only because we currently only have a single persistent volume

Reply:
It cleans all, volume(s), networks...

Reply:
Ok the purpose here, was only to reset the database